docs/deployments/predictors.md

```diff
@@ -362,6 +362,54 @@ The pre-installed system packages are listed in [images/onnx-predictor-cpu/Docke
 
 If your application requires additional dependencies, you can install additional [Python packages](python-packages.md) and [system packages](system-packages.md).
 
+## API requests
```

Comment:
These examples are really helpful. Under each predictor section, the `payload` types are already explained:
```
The `payload` parameter is parsed according to the `Content-Type` header in the request. For `Content-Type: application/json`, `payload` will be the parsed JSON body. For `Content-Type: multipart/form` or `Content-Type: application/x-www-form-urlencoded`, `payload` will be `starlette.datastructures.FormData` (key-value pairs where the value is a `string` for form data, or `starlette.datastructures.UploadFile` for file uploads, see [Starlette's documentation](https://www.starlette.io/requests/#request-files)). For all other `Content-Type` values, `payload` will be the raw `bytes` of the request body.
```
Maybe we can make that paragraph shorter and have a link to this section. Then under the API Requests we can have a little paragraph reiterating some of the various payload types and then point to these examples.

Reply:
I reordered them differently a bit. Let me know what you think!